tests/testutils/httputils.go

```diff
@@ -44,6 +43,8 @@ func HttpServe(addr string, timeout int) error {
 
 	c := make(chan string)
 	go func() {
+		log.Printf("%v: serving on %v\n", hostname, addr)
```

Comment:
Don't want to block and this is a separate issue, please file a follow up?

On Fri, Oct 16, 2015 at 11:33 AM, Stefan Junker <notifications@github.com>
wrote:

> In tests/testutils/httputils.go
> <https://github.com/coreos/rkt/pull/1608#discussion_r42274995>:
>
> > @@ -44,6 +43,8 @@ func HttpServe(addr string, timeout int) error {
> >
> >  	c := make(chan string)
> >  	go func() {
> > +		log.Printf("%v: serving on %v\n", hostname, addr)
>
> I see your point, we'd need an interface for Logger and could then use
> whatever is passed into the utility functions. We could probably use this
> interface in several projects, not just here.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/coreos/rkt/pull/1608/files#r42274995>.
>